README.md

```diff
@@ -59,9 +59,15 @@ CmdStan path set to: {a path on your file system}
 
 If you run into trouble, consult the official [`cmdstanr`](https://mc-stan.org/cmdstanr/index.html) website for further installation guides and help.
 
-## Download this repository and install the project package (`wwinference`)
+## User install
 Once `cmdstanr` and `CmdStan` are installed, the next step is to download this repository and install the package, `wwinference`. The package provides tools for specifying and running the model, and installs other needed dependencies.
+The package can be installed directly from github by running the following within an R session:
+```R
+remotes::install_github("CDCgov/ww-inference-model")
+```
 
+If you are interested in contributing to the package, this can be done by cloning the repository:
+## Developer install: Clone this repository and install the project package (`wwinference`)
```

Comment:
```suggestion
## Install `wwinference`

Once `cmdstanr` and `CmdStan` are installed, the next step is to download this repository and install the package, `wwinference`. The package provides tools for specifying and running the model, and installs other needed dependencies.

### User install

The package can be installed directly from github by running the following within an R session:
```R
remotes::install_github("CDCgov/ww-inference-model")
```

## Developer Install

If you are interested in contributing to the package, this can be done by cloning the repository and installing locally.
```

Reply:
Github doesn't like the nested code snippets. You can click "edit." to see what I intended.